docs/samples/Microsoft.ML.Samples/Dynamic/OnnxTransform.cs

```diff
@@ -0,0 +1,121 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Transforms;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.ImageAnalytics;
+using Microsoft.ML;
+using System;
+using System.IO;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    class OnnxTransformExample
+    {
+        public static void OnnxTransformSample(string[] args)
+        {
+            // Download the squeeznet image model from ONNX model zoo, version 1.2
+            // https://github.com/onnx/models/tree/master/squeezenet
+            var model_location = @"squeezenet\model.onnx";
+
+            var env = new MLContext();
+
+            // Use the utility functions to inspect models inputs, outputs, shape and type
+            // Load the model using the OnnxModel class
+            var onnxModel = new OnnxModel(model_location);
+
+            // This model has only 1 input, so inspect 0th index for input node metadata
+            var inputSchema = onnxModel.ModelInfo.InputsInfo[0];
+            var inputName = inputSchema.Name;
+            var inputShape = inputSchema.Shape;
+
+            // Deduce image dimensions from inputShape
+            var numChannels = (int) inputShape[1];
+            var imageHeight = (int) inputShape[2];
+            var imageWidth =  (int) inputShape[3];
+
+            // Similarly, get output node metadata
+            var outputSchema = onnxModel.ModelInfo.OutputsInfo[0];
+            var outputName = outputSchema.Name;
+            var outputShape = outputSchema.Shape;
+
+            var dataFile = @"test\data\images\images.tsv";
+            var imageFolder = Path.GetDirectoryName(dataFile);
+
+            // Use Textloader to load the text file which references the images to load
+            // Preview ...
+            // banana.jpg banana
+            // hotdog.jpg hotdog
+            // tomato.jpg tomato
+            var data = TextLoader.Create(env, new TextLoader.Arguments()
+            {
+                Column = new[]
+                {
+                    new TextLoader.Column("ImagePath", DataKind.TX, 0),
+                    new TextLoader.Column("Name", DataKind.TX, 1),
+                 }
+            }, new MultiFileSource(dataFile));
+
+            // Load the images referenced in the text file
+            var images = ImageLoaderTransform.Create(env, new ImageLoaderTransform.Arguments()
```

Comment:
Similar comments here and below. The sample code should use the "recommended" way of creating estimators.

Reply:
+1
please use MlContext extensions for all the transforms.

---
In reply to: [236331345](https://github.com/dotnet/machinelearning/pull/1717#discussion_r236331345) [](ancestors = 236331345)

Reply:
Updated to the recommended way of using estimators :-)

---
In reply to: [236331345](https://github.com/dotnet/machinelearning/pull/1717#discussion_r236331345) [](ancestors = 236331345)

Reply:
Done